code/modules/clothing/head/helmet.dm

```diff
@@ -209,6 +209,21 @@
 	worn_y_dimension = 64
 	custom_price = 350
 
+/obj/item/clothing/head/helmet/boatbomber
+	name = "gnome hat"
+	desc = "For those who bomb boats."
+	worn_icon = 'icons/mob/large-worn-icons/64x64/head.dmi'
+	icon_state = "gnome_hat"
+	inhand_icon_state = "constable"
+	worn_x_dimension = 64
+	worn_y_dimension = 64
+
+/obj/item/clothing/head/helmet/chika
```

Comment:
because they're subtypes of helmets, the gnome hat and the chika hat have armor values of armor = list(MELEE = 35, BULLET = 30, LASER = 30,ENERGY = 40, BOMB = 25, BIO = 0, RAD = 0, FIRE = 50, ACID = 50, WOUND = 10).

is this intended? and if not, is it okay to leave in anyway?

Reply:
who cares about clothing pieces for the announcers lol

Reply:
ohhhhhhhhhhhhhhhhhhhhhhhhhh